### PR TITLE
[update] Nightly build xml #35

### PR DESCRIPTION
**Pull request by 810**

### CODE REVIEW

**Comment by zero-24** [comment]
You need to patch the next mirror file too and add the link to the version you can find here https://developer.joomla.org/nightly-builds.html

But first we need to bump the version in the CMS version file in staging can you open an issue / PR there?

**Comment by 810** [comment]
ok updated the url and name.

**Comment by zero-24** [comment]
That the nightly build runs over night 😜  and than we can update the update server ;)

/me add a quick doku somewhere about what needs to be done at the release time with the updater files ;)

**Comment by 810** [comment]
We don't need to bump cms version.

The latest version is 3.7.0-alpha1

So nightly builds can install every night

**Comment by zero-24** [comment]
But the nightly version you install via this update is not alpha1 it is alpha2 code ;) or better we go back to dev i need to check that tomorrow ;)

File: www/core/nightlies/next_minor_extension.xml [hunk 1]
```diff
@@ -5,10 +5,10 @@
 		<description>Joomla! CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>3.7.0-dev2</version>
+		<version>3.7.0-alpha1</version>
```
Comment (zero-24) [inline www/core/nightlies/next_minor_extension.xml]: The version needs to match the entrys in the list file

**Comment by 810** [comment]
fixed, now it should open the alpha 2, if jenkins has been updated, then we can use the nightly builds again.

**Comment by zero-24** [comment]
:+1: Thanks I check that part tomorrow

**Comment by 810** [comment]
maybe i need to change it again, if the jenkins is creating; 

Joomla_3.7.0-alpha2-dev-Update_Package.zip
i have now set to:
Joomla_3.7.0-alpha2-Update_Package.zip

But someone need to change the jenkins, or are we using some other tool for the nightly builds

**Comment by zero-24** [comment]
please see: https://github.com/810/update.joomla.org/pull/1

#### Why alpha3

We need the update server point to a newer version than the current release. As every night the update needs to be applyed again :smile:

**Comment by zero-24** [comment]
@wilsonge @rdeutz this can be pushed to the update server :+1: So everything works correct tomorrow Thanks @810 :+1:

**Comment by rdeutz** [comment]
Uploaded

**Comment by zero-24** [comment]
:+1: Thanks

**Pull request merged**